src/mappings/entities/mapBlock.ts

```diff
@@ -1,7 +1,7 @@
 import { SubstrateBlock } from '@subql/types';
 import { Block } from '../../types';
 
-export const mapBlock = async (block: SubstrateBlock): Promise<Block> => {
```

Comment:
this doesn't make a detectable difference, but I figure non promise is faster than promise